packages/filebrowser/src/listing.ts

```diff
@@ -2736,14 +2747,17 @@ export namespace DirListing {
     /**
      * Create a node for a header item.
      */
-    protected createHeaderItemNode(label: string): HTMLElement {
+    protected createHeaderItemNode(label: string, title?: string): HTMLElement {
```

Comment:
The `title` attribute gets added, but I don't see it being used. Did you mean to add title on "Modified"?

Reply:
I created this new parameter when I shortened the column header all the way to "Mod", but that was very unpopular, so I have reverted this change.